docs/application/native/guides/app-management/event.md

```diff
@@ -121,7 +121,7 @@ To subscribe to a predefined system event or user-defined event:
      When defining an event name for a user event (such as `event.org.tizen.senderapp.user_event`), the name format is `event.{sender appid}.{user-defined name}`. The `{user-defined name}` must:
 
      - Contain only the ASCII characters "[A-Z][a-z][0-9]_" and not begin with a digit.
-     - Not contain the '.' (period) character.
+     - Not contain the '.'" (period) character.
```

Comment:
I don't understand this change.. Would you give me explanation?

Reply:
Done